internal/cmd/resources/resources_test.go

```diff
@@ -30,22 +30,26 @@ import (
 
 func TestPrintServicesList(t *testing.T) {
 	testCases := map[string]struct {
-		testServer *httptest.Server
-		projectID  string
-		err        bool
+		testServer   *httptest.Server
+		resourceType string
+		projectID    string
+		err          bool
 	}{
-		"list service with success": {
-			testServer: testServer(t),
-			projectID:  "found",
+		"list services with success": {
+			testServer:   testServer(t),
+			projectID:    "found",
+			resourceType: ServicesResourceType,
 		},
-		"list service with empty response": {
-			testServer: testServer(t),
-			projectID:  "empty",
+		"list deployments with empty response": {
+			testServer:   testServer(t),
+			projectID:    "empty",
+			resourceType: DeploymentResourceType,
 		},
 		"failed request": {
-			testServer: testServer(t),
-			projectID:  "fail",
-			err:        true,
+			testServer:   testServer(t),
+			projectID:    "fail",
+			err:          true,
+			resourceType: PodsResourceType,
 		},
```

Comment:
Why only tests for services and deployments? Should we add test for all resources, or otherwise the same resource in all cases?

Reply:
They are two resources chosen at random. The gist of this test, is only to check that the function will not return an error in case of a valid or empty response.

I can add every resources only for the case of valid response to excise all the cases in the switch but I think that the code contained in it is trivial.

Reply:
Done